source/common/rds/basic/config_traits_impl.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <memory>
+
+#include "envoy/rds/config_traits.h"
+
+namespace Envoy {
+namespace Rds {
+namespace Basic {
+
+/**
+ * Basic implementation of ConfigTraits interface. It can be used in simple protocols where no
+ * other parameters are required.
```

Comment:
Since this is used in both core RDS and TRDS, where else would we use something beyond this?

Reply:
Yes, in TRDS and hopefully most of other xRDS in future.
In http RDS it is not used since there are extra parameters required by `ConfigImpl` like `optional_http_filters`, `validator`, etc. There is another implementation.